js-miniapp-sdk/README.md

```diff
@@ -169,7 +164,7 @@ You can retrieve the User Name and Profile Photo of the user using the following
 ```javascript
 const info = { content: inputValue };
 
-MiniApp.user.getUserName()
+miniApp.user.getUserName()
```

Comment:
I don't know why this is changed to the camelcase. Any reason?

Reply:
To keep the consistency. We have declared this `import miniApp from "js-miniapp-sdk";` in the beginning of the document and used `miniApp` in other places as well.